### PR TITLE
added notification alert [fixes SCI-1566]

### PR DESCRIPTION
**Pull request by ZmagoD**
https://biosistemika.atlassian.net/browse/SCI-1566
used polling for fetch notification since we don't need to be listed in real time

### CODE REVIEW

File: app/controllers/client_api/notifications_controller.rb [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+      UserNotification.seen_by_user(current_user)
     end
 
-    private
-
-    def last_notifications
-      @recent_notifications =
-        UserNotification.recent_notifications(current_user)
-      UserNotification.seen_by_user(current_user)
+    def unreaded_notifications_number
```
Comment (Ducz0r) [inline app/controllers/client_api/notifications_controller.rb]: If I can be a Grammar nazi 🙂, past tense of `read` is also `read`, so please change this to `unread_notifications_number`, or even better, `unread_notifications_count` sounds better to me 🥇.

File: app/javascript/src/components/Navigation/components/NotificationsDropdown.jsx [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import React, { Component } from "react";
 import { NavDropdown } from "react-bootstrap";
 import { FormattedMessage } from "react-intl";
-import axios from "axios";
 import styled from "styled-components";
 
-import { RECENT_NOTIFICATIONS_PATH } from "../../../config/routes";
+import {
+  getRecentNotifications,
+  getUnreadedNotificationsNumber
```
Comment (Ducz0r) [inline app/javascript/src/components/Navigation/components/NotificationsDropdown.jsx]: Same here, `getUnreadNotificationsCount` maybe?

File: app/javascript/src/services/api/endpoints.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+// notifications
+export const RECENT_NOTIFICATIONS_PATH = "/client_api/recent_notifications";
+export const UNREADED_NOTIFICATIONS_PATH =
+  "/client_api/unreaded_notifications_number";
```
Comment (Ducz0r) [inline app/javascript/src/services/api/endpoints.js]: Also here ^^.

File: config/routes.rb [hunk 4]
```diff
@@ -27,6 +27,8 @@
       end
       # notifications
       get '/recent_notifications', to: 'notifications#recent_notifications'
+      get '/unreaded_notifications_number',
```
Comment (Ducz0r) [inline config/routes.rb]: And here...

File: app/javascript/src/services/api/notifications_api.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import { axiosInstance } from "./config";
```
Comment (Ducz0r) [inline app/javascript/src/services/api/notifications_api.js]: Also in this file...

File: package.json [hunk 6]
```diff
@@ -68,6 +68,11 @@
     "react-bootstrap-table": "^4.0.0",
     "react-bootstrap-timezone-picker": "^1.0.11",
     "react-data-grid": "^2.0.2",
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (ZmagoD) [reply package.json]: :banana:

**Pull request merged**